### DISCUSSION TITLE
Gradients in theme ?

### DISCUSSION THREAD

**Discussion by heresy0-0y** [chakra-ui/chakra-ui#6183] [Q&A]
Is it possible to add gradients to theme an extended theme? I've tried adding the gradients i use as the default background to colors under 'brand' but then realized the bgGradient prop doesnt take tokens from the "colors" object.

**Comment by arnoldsandoval** [comment]
Im solving this by using `layerStyles` https://chakra-ui.com/docs/styled-system/text-and-layer-styles#layer-styles

I hope this helps! 

![CleanShot 2022-06-24 at 19 31 20](https://user-images.githubusercontent.com/723659/175754910-f9743024-8fbb-4d82-88c9-e56adaec7f24.png)
